app/pages/notifications/notification-section.jsx

```diff
@@ -31,15 +31,25 @@ export default class NotificationSection extends Component {
         name: 'Zooniverse'
       });
     } else {
-      apiClient.type('projects').get(this.props.projectID)
+      apiClient.type('projects').get(this.props.projectID, { include: 'avatar' })
       .catch((error) => {
         this.setState({ error });
       })
       .then((project) => {
-        this.setState({
-          name: project.display_name,
-          avatar: project.avatar_src
-        });
+        if (project.links.avatar) {
+          apiClient.type('avatars').get(project.links.avatar.id)
+          .then((avatar) => {
```

Comment:
[minor] Does this need a `.catch(() => { this.setState({ name: project.display_name }) })` as a safety?

Reply:
Good question. The avatar should be preloaded in the client cache by the include, and if it doesn't exist then the else clause should catch that case, so in theory the get can't fail. I haven't tested that on a project that doesn't have an avatar.